tools/revng-verify-abi/Verify.cpp

```diff
@@ -0,0 +1,381 @@
+/// \file Verify.cpp
```

Comment:
This is just for testing purposes, right?
Push every executable for testing purposes in `tests/`.

Reply:
Done.